spec/reek/smells/nested_iterators_spec.rb

```diff
@@ -43,7 +75,7 @@ def chained
         @sig.keys.sort_by { |xray| xray.to_s }.each { |min| md5 << min.to_s }
       end
     EOS
-    expect(src).not_to reek_of(:NestedIterators)
+    expect(src).not_to reek_of(described_class)
```

Comment:
I also have a slight preference for the explicit `expect(src).not_to reek_of(:NestedIterators)` – especially in this particular example the _very_ English-like flow of RSpec has the most value, you just read the sentence and don’t have to do the (granted, tiny) mental jump of dereferencing `described_class`. But it’s a slight preference and I’ll be fine with `described_class` everywhere.